data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+	description "Now that you have secured the Star Queen, return to <destination> with it intact."
+	destination Valhalla
+	to offer
+		has "Bactrian Unlock [1]: done"
+	on offer
+		conversation
+			`The remaining fleet quickly enters the atmosphere of <planet> and begins scanning the surface. After about an hour of flying, you get a transmission that the ship has been found along with the location of the Star Queen.`
+			`	You watch from afar as the Corvette lands near a hangar in some sort of pirate facility. The Corvette immediately unloads what looks to be at least four dozen armed soldiers who quickly surround and breach the hangar. After about ten minutes, the hangar doors open and out rolls the Star Queen, still intact. You hear chatter over the radio saying that most of the crew and scientists are still inside.`
+			`	"Good job, everyone," you hear Lieutenant Paris say over the comms. "Now we just need to make it back to <destination> in one piece." Just as he says that and the Star Queen and Corvette begin activating their repulsor engines, a beam of light comes flying from the sky and cuts the Corvette in half. Soldiers begin scrambling into the Star Queen, which quickly takes off into space along with the rest of the fleet. You turn your repulsor engines to full and begin flying up into space behind them.`
+				launch
+	
+	on enter
+		dialog "As your repulsor engines roar, you notice a large pirate ship looming over you from space. Even from the underside, it is undeniably a Bactrian, but as to how a pirate could ever got ahold of one is a mystery."
+		dialog `	You receive a transmission from Lieutenant Paris. "I don't know how they have a Bactrian, but I suggest we jump out of here immediately and make it back to the Deep alive."`
+	
+	npc save escort accompany
+		personality timid escort
+		government Republic
+		ship "Star Queen" "D.S.S. Alcubierre"
+	
+	on visit
+		dialog `You have landed on <planet>, but you left the <npc> behind! Wait for it to enter the system before landing.`
+	
+	on complete
+		payment 100000
+
+mission "Bactrian Unlock [2]: Pirate Bactrian"
+	landing
+	invisible
+	destination Valhalla
+	to offer
+		has "Bactrian Unlock [1]: done"
+	npc kill
+		government Pirate
+		personality staying nemesis waiting
+		ship "Bactrian" "Cosmic Devil"
+		dialog `The Bactrian "<npc>" has been destroyed. Now the pirates here will have a lot harder time attacking the Republic.`
+
+mission "Bactrian Unlock [2]: More Pirates"
+	landing
+	invisible
+	to offer
+		has "Bactrian Unlock [1]: done"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		government Pirate
+		personality staying nemesis waiting
+		ship "Leviathan (Laser)" "Eve"
+		ship "Leviathan (Laser)" "Cain"
+
+mission "Bactrian Unlock [3]"
+	name "Take down the Devil"
+	description "You have returned the scientists, but the pirates still have a Bactrian. Completely destroy it and return to <destination>."
```

Comment:
"Completely destroy it" seems weird: you can't partially destroy it, so maybe make it "Destroy it"?